src/main/scala/cognite/spark/v1/DataModelInstancesRelation.scala

```diff
@@ -89,6 +89,52 @@ class DataModelInstanceRelation(config: RelationConfig, modelExternalId: String)
     )
   }
 
+  // scalastyle:off cyclomatic.complexity
+  def getInstanceFilter(sparkFilter: Filter): Seq[DataModelInstanceFilter] =
```

Comment:
Seems that we only ever return 0 or 1 element collection, so I think it would be better to use Option. If you need to return a collection, it would be better to use `DMIOrFilter` or `DMIAndFilter`

Reply:
Returning Seq seemed convenient to me because there are some DMI filters expecting lists, and easy to just concatenate. Do you think we should still change this?